app/views/games/show.html.erb

```diff
@@ -7,84 +7,84 @@
-    <div class="black"><%= render_piece(0, 1) %></div>
-    <div class="white"><%= render_piece(0, 2) %></div>
-    <div class="black"><%= render_piece(0, 3) %></div>
-    <div class="white"><%= render_piece(0, 4) %></div>
-    <div class="black"><%= render_piece(0, 5) %></div>
-    <div class="white"><%= render_piece(0, 6) %></div>
-    <div class="black"><%= render_piece(0, 7) %></div>
+    <div class="white"><%= link_to render_piece(0, 0), piece_path %></div>
+    <div class="black"><%= link_to render_piece(0, 1), piece_path %></div>
+    <div class="white"><%= link_to render_piece(0, 2), piece_path %></div>
+    <div class="black"><%= link_to render_piece(0, 3), piece_path %></div>
+    <div class="white"><%= link_to render_piece(0, 4), piece_path %></div>
+    <div class="black"><%= link_to render_piece(0, 5), piece_path %></div>
+    <div class="white"><%= link_to render_piece(0, 6), piece_path %></div>
+    <div class="black"><%= link_to render_piece(0, 7), piece_path %></div>
   </div>
   <div id="row_6">
-    <div class="black"><%= render_piece(1, 0) %></div>
-    <div class="white"><%= render_piece(1, 1) %></div>
-    <div class="black"><%= render_piece(1, 2) %></div>
-    <div class="white"><%= render_piece(1, 3) %></div>
-    <div class="black"><%= render_piece(1, 4) %></div>
-    <div class="white"><%= render_piece(1, 5) %></div>
-    <div class="black"><%= render_piece(1, 6) %></div>
-    <div class="white"><%= render_piece(1, 7) %></div>
+    <div class="black"><%= link_to render_piece(1, 0), piece_path %></div>
+    <div class="white"><%= link_to render_piece(1, 1), piece_path %></div>
+    <div class="black"><%= link_to render_piece(1, 2), piece_path %></div>
+    <div class="white"><%= link_to render_piece(1, 3), piece_path %></div>
+    <div class="black"><%= link_to render_piece(1, 4), piece_path %></div>
+    <div class="white"><%= link_to render_piece(1, 5), piece_path %></div>
+    <div class="black"><%= link_to render_piece(1, 6), piece_path %></div>
+    <div class="white"><%= link_to render_piece(1, 7), piece_path %></div>
   </div>
   <div id="row_5">
-    <div class="white"></div>
-    <div class="black"></div>
-    <div class="white"></div>
-    <div class="black"></div>
-    <div class="white"></div>
-    <div class="black"></div>
-    <div class="white"></div>
-    <div class="black"></div>
+    <div class="white"><%= link_to render_piece(2, 0), piece_path %></div>
+    <div class="black"><%= link_to render_piece(2, 1), piece_path %></div>
+    <div class="white"><%= link_to render_piece(2, 2), piece_path %></div>
+    <div class="black"><%= link_to render_piece(2, 3), piece_path %></div>
+    <div class="white"><%= link_to render_piece(2, 4), piece_path %></div>
+    <div class="black"><%= link_to render_piece(2, 5), piece_path %></div>
+    <div class="white"><%= link_to render_piece(2, 6), piece_path %></div>
+    <div class="black"><%= link_to render_piece(2, 7), piece_path %></div>
   </div>
```

Comment:
@mscoutermarsh I added `render_piece` helper to every square, since I realized if it's not there, we could move a piece to a square and then not see it anymore. I think ?
Then I ended up adding a `link_to` in every square since I assumed we'd want to be able to link to the `pieces#show` page IF we happen to move a piece in any of the empty squares. But now I have an ugly link in every square that's `nil`.

Reply:
You could move the `link_to` logic inside of `render_piece` - then it could be smarter and only show a link if a piece is there.

Does that help?

Reply:
Yes it does! Derek actually thought of that late last night, so that confirms our suspicion of putting it there. Thanks!